src/server/server.ts

```diff
@@ -46,6 +46,9 @@ server.listen(PORT, function () {
 });
 
 
+
+
```

Comment:
you can remove some whitespaces here to look cleaner

Reply:
done